linux/Jenkinsfile

```diff
@@ -21,13 +32,15 @@ pipeline {
         timeout(time: 2, unit: 'HOURS')
     }
     parameters {
+        choice(name: 'TYPE', choices: Types.ALL, description: 'Build either JDK, or CA Certificates\n  CA Certificates only on Debian possible (ARCH/VERSION irrelevant)')
```

Comment:
```suggestion
        choice(name: 'TYPE', choices: Types.ALL, description: 'Build either JDK, JRE or both')
```

Reply:
Of course, missed this (or got it again during my git squashing).

Reply:
However, _both_ is not possible in one go.